### DISCUSSION TITLE
RFC_READ_TABLE : OPTIONS TEXT 75 character limitation

### DISCUSSION THREAD

**Discussion by chandra-a** [SAP/PyRFC#296] [Q&A] [answered]
Hi All, i have a TEXT value that is quite long (about 400 characters, but can be longer in some cases)

how do i go around the 75 character limitation ? i tried splitting the string value into multiple rows but does not seem to work 
Also, is there any overall limitation of the length of the TEXT value

**Comment by bsrdjan** [comment, marked as answer]
There are thousands of ABAP function modules and chances are low that PyRFC community can help with questions about how to use some of them. The question is not related to PyRFC and I would suggest to post the question about RFC_READ_TABLE function module in SAP ABAP community: https://community.sap.com/search/?ct=qa&qt=all&q=RFC_READ_TABLE
